httpserver/http_connection.cpp

```diff
@@ -420,23 +418,20 @@ void connection_t::write_response() {
     });
 }
 
-template <typename T>
-bool connection_t::parse_header(T key_list) {
-    for (const auto key : key_list) {
-        const auto it = request_.find(key);
-        if (it == request_.end()) {
-            response_.result(http::status::bad_request);
-            response_.set(http::field::content_type, "text/plain");
-            body_stream_ << "Missing field in header : " << key;
-
-            BOOST_LOG_TRIVIAL(error) << "Missing field in header : " << key;
-            return false;
-        }
-        header_[key] = it->value().to_string();
+bool connection_t::parse_header(const char* key) {
```

Comment:
We used to respond with what headers are missing, any way to make it do that again?

Reply:
Restored, with some other changes to use `body_stream_` over `response_.body()` directly.